FirelandsProject/firelands-cata
# feat: adding rbac permissions when set gm level

### Changes proposed

- Add RBAC Permissions when you set a gmlevel for account the idea is to start a migration process over the old table account_access, and migrate to rbac_account_permissions.


### Tests performed


- build test
- in game test

### How to Test the Changes:


1. download the branch
2. build it and start worldserver
3. use the command `account set gmlevel`
4. after set gm level,  check the `rbac_account_permissions` table and verify that your gm level is added to it as a permission.

---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
